server/public/styles.css

```diff
@@ -21,3 +21,100 @@
   border: 2px solid dimgrey;
   background-color: white;
 }
+
+.container {
+  text-align: center;
+}
+
+#my-canvas {
+  border: 2px solid black;
+}
+
+#paint-tools {
```

Comment:
Is `styled-components` a good choice for your project? It's meant to replace this type of component-specific CSS.